src/OrchardCore.Modules/OrchardCore.Media.Azure/Manifest.cs

```diff
@@ -1,7 +1,7 @@
 using OrchardCore.Modules.Manifest;
 
 [assembly: Module(
-    Name = "Microsoft Azure Media",
+    Name = "Azure Media",
```

Comment:
The reason I kept these and in the docs too, is because as mentioned, MS uses Azure both with and without "Microsoft", and thus "Microsoft Azure" is the full name. So, even if we use just the short form where shorter is better (like the module name and identifier), where we have the space, being more complete is better.